website/docs/reference/resource-configs/snowflake-configs.md

```diff
@@ -341,3 +341,68 @@ In the configuration format for the model SQL file:
 </File>
 
 </VersionBlock>
+
+<VersionBlock firstVersion="1.6">
+
+## Dynamic Tables
+
+[Dynamic Tables](https://docs.snowflake.com/en/user-guide/dynamic-tables-about) are Snowflake's flavor of Materialized Views. The `CREATE DYNAMIC TABLE` ([docs](https://docs.snowflake.com/en/sql-reference/sql/create-dynamic-table)) statement requires the following parameters `TARGET_LAG` and `WAREHOUSE`, so a dbt-snowflake user must also provide these.
+
+You can create a dynamic table by editing _one_ of these files:
+
+- the SQL file for your model
+- the `dbt_project.yml` configuration file
+
+The following examples create a dynamic table:
+
+<File name='models/YOUR_MODEL_NAME.sql'>
+
+```sql
+{{
+  config(
+    materialized = 'dynamic_table',
+    warehouse = 'MY_WAREHOUSE',
+    target_lag = '10 min',
```

Comment:
`10 minutes`. Supported durations are `second`, `minute`, `hour`, `day` and their plurals.

Reply:
Note, "downsteam" is also also supported for target_lag: https://docs.snowflake.com/en/user-guide/dynamic-tables-refresh#understanding-dynamic-table-lag

Reply:
Correct, `downstream` is now supported for Snowflake; however, that has not yet been added to `dbt-snowflake`.

Reply:
```suggestion
    target_lag = '10 minutes',
```